data-visualization/accessibility.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title:  Accessibility
+category: Data visualization
+published: true
+---
+
+- [Alt tags](#alt-tags)
+- [Color contrast and color blindness](#color-contrast-color-blindness)
+ {: class="toc"}
+
+
+Allowing all users to be able to comprehend charts and graphics is a key part
+of being a government agency that serves the entire American public. This is an
+interpretation of what it means for a chart to be 508 compliant and accessible,
+starting with descriptive titles (see the [Data visualization text
+  section](text.html)) and not only relying on color to connect data to its
+  meaning (see the [Data visualization color section](color.html)).
+{: class="lead-in"}
+
+<h5>Key questions</h5>
```

Comment:
Gotcha. In other pages I was referencing, the tags and ids were explciitly set. Good to know  that they aren't needed, and I can streamline the Markdown files to change them.